pages/tickets.tsx

```diff
@@ -67,6 +71,13 @@ class TicketPage extends React.Component<WithPageMetadataProps> {
       >
         <StyledContainer>
           <h1>Tickets</h1>
+          {this.props.pageMetadata.conference.IsSoldOut === SoldOutOptions.WaitList && (
+            <p className="alert alert-warning">
+              Tickets have sold out, but we are asking people to add themselves to the waitlist since it's likely we
+              will release more tickets. Tickets will be released to the waitlist on a first come fisrt served basis so
```

Comment:
Typo in the second "first"

Reply:
lol. This is what happens when I code when I'm tired and sick late at night. Thanks